bundler.d/assets.rb

```diff
@@ -1,21 +1,21 @@
 group :assets do
   gem 'ace-rails-ap', '~> 4.0.0'
-  gem 'sass-rails', '~> 3.2'
+  gem 'sass-rails'
   gem 'uglifier', '>= 1.0.3'
   gem 'execjs', '>= 1.4.0', '<2.5.0'
   gem 'jquery-rails', '2.0.3'
```

Comment:
This will need to be updated for compatibility with our SCLed packages, we have version 3.1.0 available or can package a different one.  I could also package 2.0.3.

http://projects.theforeman.org/issues/8705 tracks this.